high_card.rb

```diff
@@ -2,4 +2,67 @@
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
+cards = []
+
+ranks.each do |rank|
+  suits.each do |suit|
+    cards.push([rank,suit])
+  end
+end
+
+cards = cards.shuffle
+
 players = []
+players_cards = []
+
+puts 'How many players are playing today?'
+num_players = gets.chomp.to_i
+
+# question - is it best practice to always pass arugments to fuctions/methods
+# rather than using glabal variables (both JS and ruby)?
+# didnt get it to work without passing in arguments
```

Comment:
Unlike Javascript, external variables aren't available inside methods... *unless* you make them instance methods, like `@players`. I wouldn't recommend that unless you're actually working inside a class.